src/cairo_dependencies_graph/import_parser.py

```diff
@@ -11,8 +11,18 @@ def _visit_default(self, obj):
         # top-level code is not generated
         return obj
 
-    def visit_CodeElementFunction(self, elm: CodeElementFunction):
-        return super().visit_CodeElementFunction(elm)
-
     def visit_CodeBlock(self, elm: CodeBlock):
-        return super().visit_CodeBlock(elm)
+        return self.extract_imports(elm)
+    
+    def extract_imports(self, elm):
+        res = []
+        for x in elm.code_elements:
+            if isinstance(x.code_elm, CodeElementImport):
+                imported_items = x.code_elm.import_items
+                for item in imported_items:
+                    res.append(item.orig_identifier.name)
```

Comment:
you're exporting the name here. Shouldn't we export the path/filename instead of the imported element name? If we want to see dependencies between files.

Reply:
you right